src/TSTransformer/classes/TransformState.ts

```diff
@@ -200,11 +200,11 @@ export class TransformState {
 		return luau.property(luau.globals.TS, name);
 	}
 
-	public checkJsxFactory(node: ts.Node) {
-		this.multiTransformState.checkJsxFactory(node, this.compilerOptions);
+	public isRoactJsxFactory(node: ts.Node) {
```

Comment:
Maybe we should just invoke the MultiTransformState method directly instead of having these proxy functions?

Reply:
I feel like this adds more duplication, because of passing compilerOptions. it's not great though :/

Reply:
We could give CompilerOptions to MultiTransformState in it's constructor

Reply:
That would be duplicating the object/reference. But I guess it's immutable so that's fine?

Reply:
Now that I think about it, it could also use the new `addDiagnosticFromNodeIfNotCached`. No transformstate call necessary